README.md

```diff
@@ -43,10 +43,6 @@ for developing state-of-the-art deep learning models on a wide variety of lingui
     <td><b> allennlp.nn </b></td>
     <td> tensor utility functions, such as initializers and activation functions </td>
 </tr>
-<tr>
```

Comment:
Maybe point to the external repo?

More generally, somewhere in our README we should have a list of all of our related subrepos and what's in them.

Reply:
This is where it would be really handy to have an `allennlp` organisation on github - Github provides us a straightforward way to make these discoverable without having to reference them everywhere... We probably aren't going to get that though, I assume. I'll leave this for once we've done the rest of the sub-repos.

Reply:
@schmmd, what do you think about this?

Reply:
Making a github.com/allennlp?  Since we need private repos it sounds like a headache until we're a 5013c.